com.unity.postprocessing/PostProcessing/Shaders/Builtins/MultiScaleVORender.compute

```diff
@@ -80,20 +80,21 @@ CBUFFER_START(CB1)
     float4 gInvThicknessTable[3];
     float4 gSampleWeightTable[3];
     float4 gInvSliceDimension;
-    float2 AdditionalParams;
+    float3 AdditionalParams;
 CBUFFER_END
 
 #define gRejectFadeoff AdditionalParams.x
 #define gIntensity AdditionalParams.y
+#define zBias AdditionalParams.z
 
 #ifdef MSAA
 float2 TestSamplePair(float frontDepth, float2 invRange, uint base, int offset)
 {
     // "Disocclusion" measures the penetration distance of the depth sample within the sphere.
     // Disocclusion < 0 (full occlusion) -> the sample fell in front of the sphere
     // Disocclusion > 1 (no occlusion) -> the sample fell behind the sphere
-    float2 disocclusion1 = DepthSamples[base + offset] * invRange - frontDepth;
-    float2 disocclusion2 = DepthSamples[base - offset] * invRange - frontDepth;
+    float2 disocclusion1 = DepthSamples[base + offset] * invRange - (frontDepth - zBias);
```

Comment:
frontDepth is related to invThickness of sphere samples, which isn't what we want. Z bias should be done on the screen sample in Main() before invThisDepth is calculated. Remember, thisIdx in Main() is the index of the screen sample where the sphere samples are centered around.